umbraco-cloud/upgrades/version-specific-upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,161 @@
+---
+description: This article will provide detailed steps on how to migrate a Umbraco 7 Cloud project to Umbraco 8.
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+Taking your Umbraco CMS project from Umbraco 7 to 8 is called a migration as it requires that the data is migrated in the process. This article covers each step involved in this process.
+
+Read the [general article about Content migration](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md#what-are-the-limitations) to learn more about limitations and other things related to migrating your Umbraco site from 7 to 8.
+
+## Video tutorial
+
+<iframe width="800" height="450" title="Migrate an Umbraco Cloud project from 7 to 8: Introduction" src="https://www.youtube.com/embed/wNIDdgdAt8s?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+You can find the full playlist here: [Migrate an Umbraco Cloud project from 7 to 8](https://www.youtube.com/playlist?list=PLgX62vUaGZsGceCXveMkprlDV5a3K94db)
+
+## Prerequisites
+
+* A Umbraco 7 Cloud project running **the latest version of Umbraco 7**.
+* Make sure Umbraco Forms data is not handled as content.
+  * See [Umbraco Forms on Cloud](../../deployment/umbraco-forms-on-cloud.md#how-forms-are-handled-on-umbraco-cloud) for more details on how to check the setting.
+* A clean Cloud project running the latest version of Umbraco 8 with **at least 2 environments**.
+
+{% hint style="info" %}
+We strongly recommend having at least 2 environments on the Umbraco 8 project.
+
+Should something fail during the migration, the Development environment can always be removed and re-added in order to start over on a clean slate.
+{% endhint %}
+
+## Step 1: Content migration
+
+1. Clone down the Umbraco 7 Cloud project.
+2. Run the project locally and **restore** Content and Media.
+3. Clone down the Development environment from the Umbraco 8 Cloud project.
```

Comment:
I think it would be helpful to add a tip or note linking to this article: https://github.com/umbraco/UmbracoDocs/blob/legacy-docs/Umbraco-Cloud/Set-Up/Working-Locally/Working-With-Visual-Studio/index.md (in case, someone doesn't know how to generate a .csproj file).

Also, I can find the updated version of this article on the docs site, not the v7-v8 version.

Reply:
Using the UaaS.cmd tool is an outdated way of working with Cloud, so I do not think we should link to it.
Also, it's not something that we recommend as the tool isn't being maintained by HQ anymore.

We could add a link to the tool to this article instead, perhaps 🤔 
With a few notes on how to use it, and with the message that this is only a tool to be used for this purpose...

Reply:
Yup, some tips/info around creating a .csproj will be helpful considering the next step to install a package requires one. It may or may not be useful but it can be helpful to just put it out there 😀

Reply:
I added a note between bullet 3 and 4.
Let me know what you think!

Reply:
Looks good to me 🙌